src/UserSettingsPage.cpp

```diff
@@ -687,7 +687,7 @@ UserSettingsPage::UserSettingsPage(QSharedPointer<UserSettings> settings, QWidge
         boxWrap(tr("Device ID"), deviceIdValue_);
         boxWrap(tr("Device Fingerprint"), deviceFingerprintValue_);
         boxWrap(
-          tr("Share keys with trusted users"),
+          tr("Share keys with verified users"),
```

Comment:
I guess that does make it clearer yes. We should use a consistent vocabulary!

Maybe make that "Share keys with verified users and devices", since this also shares it with devices, when you only did device verification without cross-signing, although it may be too specific?

Reply:
I like it. It's more accurate to what is actually happening!